index.bs

```diff
@@ -3503,6 +3503,15 @@ the following algorithm returns <i>true</i>.
             <a href="#limit-bigint-numeric-overloading">a separate restriction on their use in
             overloading</a> below. Please also note <a href="#limit-bigint-numeric-unions">the
             advice about using unions of these types</a>.
+        1.  A [=callback function=] that does not have
+            [{{LegacyTreatNonObjectAsNull}}] extended attribute is
+            distinguishable from a dictionary-like category object based on
+            whether the object is [=ECMAScript/callable=]. When selecting a
+            type for the purposes of [=overload resolution algorithm=] or
+            [=union type=] selection, and the possible types are [=callback
+            function=] and disctionary-like category type, then the type is a
+            [=callback function=] if the object is [=ECMAScript/callable=].
+            Otherwise, it is a dictionary-like category type.
```

Comment:
For clarity, even though this pull request doesn't change it, the overload resolution algorithm and algorithm "convert an ECMAScript value an to IDL union type" already handle this correctly in the Callable(V) case, as I understand it?